docs/release-notes/toc.yml

```diff
@@ -0,0 +1,5 @@
+toc:
+  - file: index.md
```

Comment:
Something is wrong with rendering

![Schermata del 2025-03-04 17-11-02](https://github.com/user-attachments/assets/b93b52cf-ce71-4c16-aa9f-209ee670bbfc)

Reply:
This is expected for now. It will be resolved when we're able to bundle pages from all repos together. https://github.com/elastic/docs-builder/issues/621